ui/StatusQ/src/StatusQ/Components/StatusTimeStampLabel.qml

```diff
@@ -12,7 +12,24 @@ StatusBaseText {
     color: Theme.palette.baseColor1
     font.pixelSize: 10
     visible: !!text
-    text: showFullTimestamp ? LocaleUtils.formatDateTime(timestamp) : LocaleUtils.formatRelativeTimestamp(timestamp)
+    text: d.formattedLabel
+
+    QtObject {
+        id: d
+        property string formattedLabel: root.showFullTimestamp ? LocaleUtils.formatDateTime(root.timestamp) : LocaleUtils.formatRelativeTimestamp(root.timestamp)
+
+        readonly property var _conn: Connections {
+            target: StatusSharedUpdateTimer
+            enabled: root.visible && root.timestamp && !root.showFullTimestamp
+            function onTriggered() {
+                const newFormattedLabel = LocaleUtils.formatRelativeTimestamp(root.timestamp)
+                if (newFormattedLabel !== d.formattedLabel) {
+                    d.formattedLabel = newFormattedLabel
```

Comment:
It overwrites the initial binding. So when `showFullTimestamp` goes to true, `formattedLabel` will be stuck with outdated relative timestamp forever :thinking: Right?

Reply:
In theory yes but we never change that property

Reply:
Fixed